package.json

```diff
@@ -65,5 +65,8 @@
         "tailwindcss": "^3.2.4",
         "tslib": "^2.4.0",
         "typescript": "^4.8.4"
+    },
+    "publishConfig": {
+        "registry": "https://registry.npmjs.com"
```

Comment:
I added this for those of us who have different default registries. I hope you don't mind. Let me know if you would like to remove it.

Reply:
I think it's better to leave this as it is. You can just add your default registry when you publish from your fork

Reply:
Sure. Will remove!